android/app/src/main/res/values/styles.xml

```diff
@@ -22,7 +22,7 @@
         <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
         <item name="colorAccent">@color/colorAccent</item>
         <item name="android:listDivider">@color/white_slightly_opaque</item>
-
+        <item name="questionnaire_theme">@style/Theme.MaterialComponents.Light.DarkActionBar</item>
```

Comment:
Does setting the `AppTheme` parent to `Theme.MaterialComponents.Light.DarkActionBar` work?

Reply:
No, it won't work because the Questionnaire fragment uses its own theme but we can use a custom theme by using the `questionnaire_theme` variable.

https://github.com/google/android-fhir/blob/79c7db879726fa1492578f04edb053cf8546f9c1/datacapture/src/main/java/com/google/android/fhir/datacapture/QuestionnaireFragment.kt#L40-L47